main.tf

```diff
@@ -4,29 +4,43 @@ This is an example Terraform file to show capabilities of the VCert integration.
 
 
 variable "CLOUD_APIKEY" {
+  default = ""
 }
 
 variable "CLOUD_ZONE" {
+  default = ""
 }
 
 variable "CLOUD_URL" {
   default = ""
 }
 
 variable "TPP_USER" {
+  default = ""
 }
 
 variable "TPP_PASSWORD" {
-
+  default = ""
 }
 
 variable "TPP_URL" {
+  default = ""
 }
 
 variable "TPP_ZONE" {
+  default = ""
 }
 
 variable "TRUST_BUNDLE" {
+  default = ""
+}
+
+variable "ACCESS_TOKEN" {
+  default = ""
+}
+
+variable "TPP_TOKEN_URL"{
```

Comment:
Do we need this?  This update only involves using a token so we don't need to communicate with a different URL (i.e. /VEDAuth).

Reply:
Removed unnecessary variable TPP_TOKEN_URL